src/client/context.rs

```diff
@@ -1,17 +1,16 @@
-use std::sync::Arc;
+use std::sync::{Arc, Mutex};
```

Comment:
For all the mutex changes to this import, please change it back, because serenity is meant to use parking_lot's mutex to avoid deadlocks on windows.